persistence-api/src/test/java/io/stargate/db/schema/ColumnTest.java

```diff
@@ -251,11 +253,14 @@ public void testToString() throws UnknownHostException {
   @Test
   public void testToStringWithCollections() {
     java.util.List<Integer> list = Arrays.asList(1, 2, 3);
-    Set<Integer> set = new HashSet<>(list);
+    java.util.TreeSet<Integer> set = new TreeSet(list);
+    java.util.Map<Integer, Integer> m = new TreeMap<>();
+    m.put(1, 1);
+    m.put(2, 2);
+    String mapString = Type.Map.of(Type.Int, Type.Int).toString(ImmutableMap.of(1, 1, 2, 2));
     assertThat(Type.List.of(Type.Int).toString(list)).isEqualTo("[1,2,3]");
     assertThat(Type.Set.of(Type.Int).toString(set)).isEqualTo("{1,2,3}");
-    assertThat(Type.Map.of(Type.Int, Type.Int).toString(ImmutableMap.of(1, 1, 2, 2)))
-        .isEqualTo("{1:1,2:2}");
+    assertThat(Type.Map.of(Type.Int, Type.Int).toString(m)).isEqualTo("{1:1, 2:2}");
```

Comment:
The CI failed because of:

```
Step #4 - "cassandra-3.11": [ERROR] Tests run: 27, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.551 s <<< FAILURE! - in io.stargate.db.schema.ColumnTest
Step #4 - "cassandra-3.11": [ERROR] testToStringWithCollections  Time elapsed: 0.018 s  <<< FAILURE!
Step #4 - "cassandra-3.11": org.opentest4j.AssertionFailedError: 
Step #4 - "cassandra-3.11": 
Step #4 - "cassandra-3.11": Expecting:
Step #4 - "cassandra-3.11":  <"{1:1,2:2}">
Step #4 - "cassandra-3.11": to be equal to:
Step #4 - "cassandra-3.11":  <"{1:1, 2:2}">
Step #4 - "cassandra-3.11": but was not.
Step #4 - "cassandra-3.11": 	at io.stargate.db.schema.ColumnTest.testToStringWithCollections(ColumnTest.java:263)
Step #4 - "cassandra-3.11": 
Step #4 - "cassandra-3.11": [INFO] 
Step #4 - "cassandra-3.11": [INFO] Results:
Step #4 - "cassandra-3.11": [INFO] 
Step #4 - "cassandra-3.11": [ERROR] Failures: 
Step #4 - "cassandra-3.11": [ERROR]   ColumnTest.testToStringWithCollections:263 
Step #4 - "cassandra-3.11": Expecting:
Step #4 - "cassandra-3.11":  <"{1:1,2:2}">
Step #4 - "cassandra-3.11": to be equal to:
Step #4 - "cassandra-3.11":  <"{1:1, 2:2}">
Step #4 - "cassandra-3.11": but was not.
Step #4 - "cassandra-3.11": [INFO] 
Step #4 - "cassandra-3.11": [ERROR] Tests run: 138, Failures: 1, Errors: 0, Skipped: 0
```

```suggestion
    assertThat(Type.Map.of(Type.Int, Type.Int).toString(m)).isEqualTo("{1:1,2:2}");
```